packages/jupyter-ai/src/components/chat-input.tsx

```diff
@@ -201,6 +202,20 @@ export function ChatInput(props: ChatInputProps): JSX.Element {
         selection: { ...cellWithError, type: 'cell-with-error' }
       });
       return;
+    } else if (currSlashCommand === '/learn') {
+      const prompt_split = prompt.split(' ');
+      if (prompt_split[prompt_split.length - 1] === '') {
+        showDialog({
+          title: 'No directory specified for /learn',
+          body: '/learn will create embeddings on all valid files under your root directory (or preferred directory if set.) This may take a long time.',
+          buttons: [Dialog.cancelButton(), Dialog.warnButton({ label: 'OK' })]
+        }).then(result => {
+          if (result.button.accept) {
+            props.chatHandler.sendMessage({ prompt, selection });
+          }
+        });
+      }
```

Comment:
Should there be an `else` clause sending the message otherwise?

Reply:
Yes, there should. Thanks for catching that.